user/views.py

```diff
@@ -1,3 +1,50 @@
-from django.shortcuts import render
+import json
+import bcrypt
+import jwt
+import requests
 
-# Create your views here.
+from django.views import View
+from django.http  import JsonResponse
+
+from my_settings  import SECRET_KEY, ALGORITHM
+from .models      import User, UserType, UserTier
+
+USER_TIER_DEFAULT_ID = 1
+USER_TYPE_DEFAULT_ID = 1
+
+class KakaoLoginView(View):
+    def get(self, request):
+        try:
+            access_token = request.headers["Authorization"]
+            headers      =({'Authorization' : f"Bearer {access_token}"})
+            url          = "https://kapi.kakao.com/v2/user/me"
+            response     = requests.post(url, headers=headers)
+            user         = response.json()
+
+            if user.get('msg', None):
+                return JsonResponse({'message':'INVALID_TOKEN'}, status = 400)
+
+            name  = user['properties']['nickname']
+            email = user['kakao_account'].get('email')
+
+            if User.objects.filter(social_login = str(user['id'])).exists():
```

Comment:
```suggestion
            if User.objects.filter(social_login = str(user['id'])).exists():
```
여기 `str()`으로 형변환 하지 않으면 혹시 동작에 에러가 발생하나요?

Reply:
User의 models.py를 작성할 때 당시 카톡에서 들어오는 id가 str인지 int인지 몰라서 일단 CharField로 작성해놨습니다. 결과적으로는 int가 들어왔고, 작업 편의를 위해 models.py에서 IntegerField로 바꾸는 대신 view에서 str으로 바꿔서 저장했습니다. models.py를 바꾸는 방향으로 팀원과 상의 후 결정하겠습니다.